### DISCUSSION TITLE
Enhancement of the precision of map-db

### DISCUSSION THREAD

**Discussion by sabosabo77** [stella-cv/stella_vslam#623] [Q&A]
**What I Want to Achieve**

I want to enhance the precision of the map database used in Localization mode.

Input:
	•	Camera trajectory (high-accuracy self-positioning results, such as those from LIDAR-SLAM)
	•	Monocular RGB images at each timestamp

Output:
	•	Map database

**Question**

Even with offline processing, I believe this can be achieved by calling some of the functions already implemented in Stella-VSLAM from my custom program. However, the code is complex, and I’m struggling to understand the behavior of each function.

Could you provide general guidance on which functions to call and in what order?
Additionally, if there are any documents that can help me understand Stella-VSLAM’s algorithm, I would appreciate it if you could share them.

**Comment by Skwangles** [comment]
Hi,

Edit: coming back to this thread, I realise you are asking if there is a way to combine the two for a more precise mapping. Sorry, that would best be ynd_stellas wheelhouse. 


If you want help understanding the code, Ai agents are amazing for breaking down the key interfaces and could help you. 

Otherwise, here are the original academic papers:

OpenVSLAM (Stella VSLAM's original project) t is a reimplementation of the [OrbSLAM2 paper](https://arxiv.org/abs/1610.06475). 
Here is the actual [OpenVSLAM paper](https://arxiv.org/abs/1910.01122) that Stella VSLAM has been forked from.
